pkg/cmd/mysqlout-to-csv/main.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/cockroachdb/cockroach/pkg/util"
+)
+
+var lineSep = flag.String("lines-terminated-by", "\n", "line separator character")
+var fieldsSep = flag.String("fields-terminated-by", "\t", "field separator character")
+var encloseChar = flag.String("fields-enclosed-by", "", "field enclosing character")
```

Comment:
does this also need an "is optional" flag to support `FIELDS [OPTIONALLY] ENCLOSED BY`? or maybe this parser can tell if it was enclosed or not with a lookahead or something? need to test this.